data/human/free worlds 1 start.txt

```diff
@@ -1034,7 +1034,10 @@ mission "FW Senate 1B"
 				`	"I have no idea."`
 				`	"Ummm... they both involve momentous decisions that have serious consequences?"`
 				`	"They both require someone skilled and decisive to be in command?"`
-			`	"The correct answer," she says, "was, 'nothing whatsoever; they are entirely different disciplines.' Now, hurry up, I hope to be on Bourne a week from now." You consider explaining to her that it's impossible to get there that fast, but decide against it.`
+			`	"The correct answer," she says, "was, 'nothing whatsoever; they are entirely different disciplines.' Now, hurry up, I hope to be on Bourne a week from now." You consider explaining to her that it's impossible to get there that fast using a hyperdrive, but decide against it.`
```

Comment:
This should have an extra tab of indentation.

Reply:
Wait - why does it need a tab?

Reply:
You now will only see `using a hyperdrive` if you have a Jump Drive.

Reply:
It has less indentation than the other dialogue lines.